core/templates/pages/topic-editor-page/services/topic-editor-state.service.ts

```diff
@@ -327,8 +327,6 @@ angular.module('oppia').factory('TopicEditorStateService', [
         if (index === null) {
           if (newIndex === -1) {
             return;
-          } else {
-            throw Error('Invalid subtopic page.');
```

Comment:
This is removed since, according to the comment above, this shouldn't be here at all. Also, I encountered this error when deleting a subtopic without clicking on its 'Edit' link, as then its subtopic page data wasn't loaded yet (which is what the comment above says).